environment.yml

```diff
@@ -1,8 +1,8 @@
-name: base
+name: mne
 channels:
-- defaults
+- conda-forge
 dependencies:
-- python>=3.5
+- python>=3.6
```

Comment:
(next MNE-Python release, I mean)

Reply:
Make me wonder why we share the same `environment.yml` between `master` and stable?

Reply:
I've reverted this change in dca2f71

Reply:
Also I'm actually not sure if `conda env update` even respects the environment name – according to the docs it will always install into the currently active environment, unless the `--name` command line switch is used.